### DISCUSSION TITLE
How to integrate with nvim-cokeline?

### DISCUSSION THREAD

**Discussion by traverseda** [nvim-neo-tree/neo-tree.nvim#768] [Q&A] [answered]
nvim-cokeline has [sidebar support](https://github.com/noib3/nvim-cokeline#sidebars), but it doesn't seem to work for me when I try to replace `NvimTree` with `Neotree`.

Does anyone have a working example?

**Comment by miversen33** [comment, marked as answer]
The filetype is `neo-tree`
```
:set filetype?
  filetype=neo-tree
```

![image](https://user-images.githubusercontent.com/2640668/221043884-1a87f7ce-cece-41f0-b9cd-d027135bd062.png)

Here is a (_really_) shitty sidebar component for cokeline
```lua
sidebar = {
    filetype = 'neo-tree',
    components = {
    {
            text = "    Neo-tree",
            fg = vim.g.terminal_color_3,
            bg = get_hex("NeoTreeNormal", 'bg'),
            style = 'bold'
    }}
}
```

Also thanks for showing me this, I didn't know cokeline had that configuration option :smile:

**Comment by traverseda** [comment]
Thanks, I appreciate the help. It's my first time getting really in-depth with vim plugins, previously I've been using pretty much the defaults with a few very simple plugin additions.
